src/components/z-app-header/index.tsx

```diff
@@ -78,33 +40,10 @@ export class ZAppHeader {
   stuck = false;
 
   /**
-   * Set the hero image source for the header.
-   * You can also use a [slot="hero"] node for advanced customization.
+   * the menu bar is not displayed and a burger icon appears to open the offcanvas menu
    */
-  @Prop()
-  hero: string;
-
-  /**
-   * Should place an overlay over the hero image.
-   * Useful for legibility purpose.
-   */
-  @Prop({reflect: true})
-  overlay = false;
-
-  /**
-   * Control menu bar position in the header.
-   * - auto: the menu bar is positioned near the title
-   * - stack: the menu bar is positioned below the title
-   * - offcanvas: the menu bar is not displayed and a burger icon appears to open the offcanvas menu
-   */
-  @Prop({reflect: true})
-  flow: "auto" | "stack" | "offcanvas" = "auto";
-
-  /**
-   * The opening state of the drawer.
-   */
-  @Prop({reflect: true})
-  drawerOpen = false;
+  @Prop({reflect: true, mutable: true})
```

Comment:
Non so se sia una bella cosa poter mutare i dati in ingresso, se ti ricordi con me quella volta per lo userId, che sicuramente era un dato molto piu' critico, ma non potremmo fare la stessa cosa gestendola con tipo lo stato del componente?

Reply:
cambiare la prop internamente nasce dall'esigenza specifica di verificare che la dimensione del menu non superi quella della viewport, in questo modo abilitò dall'interno il cambiamento.